core/trino-main/src/main/java/io/trino/execution/buffer/OutputBufferMemoryManager.java

```diff
@@ -199,6 +224,7 @@ public synchronized void close()
             memoryContext.close();
         }
         closed = true;
+        recordOutputBufferUtilization();
```

Comment:
this is probably redundent as there is `updateMemoryUsage(-bufferedBytes.get());` above

Reply:
Yes, you right